packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -980,7 +980,10 @@ export function toSvgFilters( tree, blockSelectors ) {
 }
 
 const getSelectorsConfig = ( blockType, rootSelector ) => {
-	if ( ! isEmpty( blockType?.selectors ) ) {
+	if (
+		blockType?.selectors &&
+		Object.keys( blockType?.selectors ).length > 0
```

Comment:
Selectors, if defined, will be an object:

https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#selectors

Reply:
Super nit: `?` in `blockType?.selector` at the second check is not needed.

Reply:
Indeed, good find!

Cleaned it up in https://github.com/WordPress/gutenberg/pull/51098/commits/262c8db2a1cf33721af3e6cb7efa435b40e97ee1